.appveyor.yml

```diff
@@ -32,3 +32,4 @@ build_script:
   - ps: |
       cmake --build . 2>&1 | Out-File build_output.txt
       Push-AppveyorArtifact .\build_output.txt
+      if($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode )  }
```

Comment:
Isn't this checking the error code of `Push-AppveyorArtifact`?

Reply:
That's correct. Ugh.